<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchBox</class>
 <widget class="QGroupBox" name="searchBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>437</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>416</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GroupBox</string>
  </property>
  <property name="title">
   <string>Search</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>551</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout" stretch="0,0">
    <item>
     <layout class="QVBoxLayout" name="searchItems" stretch="">
      <property name="spacing">
       <number>-1</number>
      </property>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="runSearch">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>verticalLayoutWidget_2</zorder>
  <zorder>runSearch</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
